package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "canvg",
+  "name": "canvg-origin",
```

Comment:
@graingert what was the point of the rename here again?

Reply:
canvg is currently being squatted by node-canvg

Reply:
who owns that / can we reach out  and convince them to rename?

Reply:
@yetzt (https://github.com/yetzt/node-canvg)

Reply:
@gabelerner if @yetzt were to hand over the name it would break everyone using node-canvg (via canvg)

Reply:
@graingert it's not squatted, see #425
> i'm happily signing the npm package over to @gabelerner
this still applies

Reply:
>  i said i'd be willing if the official package would maintain the same apis, work in nodejs and not break any package dependents; they told me they'd merge at some point and reach out to me when they're ready. i heard nothing since.

Reply:
@yetzt it would be good to get your stuff merged in, but I'm more interested in getting this repo published to npm.

Reply:
@gabelerner what about `@canvg/browser` instead of `canvg-origin`?

Reply:
...or `@canvg/canvg`?

Reply:
don't mind I just want this up-to-date on npm!

Reply:
`@canvg/server` and `@canvg/browser` makes sense to me. If @yetzt and @gabelerner are both up for that I'll hand the names over.

If anyone can pick better names I'll use those instead!

Reply:
So I like PR https://github.com/canvg/canvg/pull/444 ... the snapshot testing done there would be nice in preventing future breaking changes.  prior to that i manually spot checked things.

Just to remove any of my confusion, if that gets merged, then the repo will be working on node, and then we can get the name `canvg` from @yetzt and @graingert is happy since this is now updated?

Reply:
SGTM

Reply:
@gabelerner i presume you are `gabelerner` on npm as well? i'll put you in as owner. :)

Reply:
Yup thanks

Reply:
@gabelerner ok I've rolled back the change to the package name.

Can we get a release of 1.0.0, please?

Reply:
@graingert published current 1.0.0 version